components/layout_thread/lib.rs

```diff
@@ -467,18 +463,6 @@ impl Layout for LayoutThread {
     }
 
     fn exit_now(&mut self) {
-        // Drop the root flow explicitly to avoid holding style data, such as
```

Comment:
Looks like the code this comment applies to still exists, so let's revert the comment deletion.

Reply:
Oh, thanks for spotting this. Tangentially, this is something that doesn't happen in Layout 2020, so I suspect that we do not need it for legacy layout.